generator/common/Makefile

```diff
@@ -11,23 +11,36 @@ PREPARE_COMMAND?=
 test: prepare-test
 	. ${PYTHON_ENV}/bin/activate; \
 	export GOPATH=${PWD}/build ; \
-	export PATH=${PATH}:${PWD}/build/bin; \
+	export PATH=$${GOPATH}/bin:${PATH}; \
```

Comment:
`$$` on purpose?

Reply:
Yes. It's an escape so that GOPATH is resolve in the shell command rather than by Make. If you don't do this then you get the original GOPATH value and not the one set 2 lines above.